swo_aws_extension/flows/steps/create_subscription.py

```
@@ -32,6 +38,16 @@ def pre_step(self, context: PurchaseContext) -> None:
                 f" is not '{PhasesEnum.CREATE_SUBSCRIPTION}'"
             )
+        if (
+            get_customer_roles_deployed(context.order) == CustomerRolesDeployed.NO_DEPLOYED
+            and get_support_type(context.order) == SupportTypesEnum.PARTNER_LED_SUPPORT
+        ):
+            raise ConfigurationStepError(
+                "Action Required: Order must be failed manually",
+                f"{context.order_id} - Error - The order has been flagged to fail on the "
+                f"marketplace because the customer are not deployed the roles required for "
+                f"PLS support.",
+            )
```

Ruff (0.14.14)

Avoid specifying long messages outside the exception class

(TRY003)